### PR TITLE
Config updates for first release

### PR DESCRIPTION
**Pull request by dwillmer**
- Updated minimum test coverage to 99%
- Removed 'Alpha' flag from pyPI classifiers
- Updated minimum numba version to 0.37.0 (it fixes lots of threading bugs from 0.36)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def read_utf8(filename):
     # For a list of valid classifiers, see
     # https://pypi.python.org/pypi?%3Aaction=list_classifiers
     classifiers=[
-        'Development Status :: 3 - Alpha',
```
Comment (pawroman) [inline setup.py:47]: I think you are looking for `Development Status :: 4 - Beta`, vide list of valid classifiers: https://pypi.python.org/pypi?%3Aaction=list_classifiers

File: pytest.ini [hunk 2]
```diff
@@ -1,4 +1,4 @@
 [pytest]
-addopts = --doctest-modules --doctest-glob='docs/source/*.rst' --cov . --cov-report term-missing --durations=5
-testpaths = fastats tests
+addopts = --doctest-modules --doctest-glob='docs/source/*.rst' --cov . --cov-report term-missing --durations=10
+testpaths = fastats tests docs
```
Comment (pawroman) [inline pytest.ini:3]: Nice one, doctests in docs rule!

File: setup.py (hunk 1 above)
Comment (dwillmer) [reply setup.py:47]: My bad, thanks!

File: pytest.ini (hunk 2 above)
Comment (dwillmer) [reply pytest.ini:3]: 👍

**Comment by rjenc29** [comment]
Looks good to me - nice one

**Pull request merged**